commands/bootstrap/__tests__/bootstrap-command.test.js

```diff
@@ -345,6 +345,14 @@ describe("BootstrapCommand", () => {
       });
     });
 
+    it("should respect --contents argument during linking step", async () => {
```

Comment:
@evocateur You're right, I thought I was using the _with-contents_ fixture for this test but I wasn't! This was meant to test the new `--contents` argument I added.

Should be fixed now.